src/EditorFeatures/Core/Extensibility/QuickInfo/IQuickInfoPresenterSession.cs

```diff
@@ -1,12 +1,13 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using Microsoft.CodeAnalysis.QuickInfo;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Language.Intellisense;
 using Microsoft.VisualStudio.Text;
 
 namespace Microsoft.CodeAnalysis.Editor
 {
     internal interface IQuickInfoPresenterSession : IIntelliSensePresenterSession
```

Comment:
Now that editor owns presentation of quick info and quick info API is called on non-UI thread, is this still needed? The notion of a Roslyn quick info presenter feels vestigial.

Reply:
Actually, I don't see any implementers of this interface anywhere...

Reply:
deleted